src/OnlineSales/Services/ContactService.cs

```diff
@@ -0,0 +1,70 @@
+﻿// <copyright file="ContactService.cs" company="WavePoint Co. Ltd.">
+// Licensed under the MIT license. See LICENSE file in the samples root for full license information.
+// </copyright>
+
+using OnlineSales.Data;
+using OnlineSales.Entities;
+using OnlineSales.Interfaces;
+
+namespace OnlineSales.Services
+{
+    public class ContactService : IContactService
+    {
+        private readonly ApiDbContext apiDbContext;
+
+        public ContactService(ApiDbContext apiDbContext)
+        {
+            this.apiDbContext = apiDbContext;
+        }
+
+        public async Task<Contact> AddContact(Contact contact)
+        {
+            var contactWrappWithList = new List<Contact>()
+            { contact };
+
+            var returnedValue = DomainMapperWithContacts(contactWrappWithList);
+
+            await apiDbContext.Contacts!.AddRangeAsync(returnedValue);
+
+            await apiDbContext.SaveChangesAsync();
+
+            return returnedValue!.FirstOrDefault() !;
```

Comment:
@WASandaruwan 

1) Why not to simple return a "contact" value here?
2) Please get rid of  new List<Contact>() { contact } by using an overload method which accept a single parameter.